### ISSUE TITLE
Reversal of factor levels in vignette example changes data

### ISSUE THREAD

**Issue by jtr13** [corybrunson/ggalluvial#66]
Thank you for a great package.

I'd like to point out that the line `levels(vaccinations$response) <- rev(levels(vaccinations$response))` in the package vignette erroneously relabels the factor levels:

``` r
library(ggalluvial)
#> Loading required package: ggplot2
    data(vaccinations)
    summary(vaccinations$response)
#>   Missing     Never Sometimes    Always 
#>        26        25        37        29
    levels(vaccinations$response) <- rev(levels(vaccinations$response))
    summary(vaccinations$response)
#>    Always Sometimes     Never   Missing 
#>        26        25        37        29
```

<sup>Created on 2020-10-14 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

In order to keep the levels as they are and only change their order, it should be:

``` r
library(ggalluvial)
#> Loading required package: ggplot2
    data(vaccinations)
    summary(vaccinations$response)
#>   Missing     Never Sometimes    Always 
#>        26        25        37        29
    vaccinations$response <- factor(vaccinations$response, levels = rev(levels(vaccinations$response)))
    summary(vaccinations$response)
#>    Always Sometimes     Never   Missing 
#>        29        37        25        26
```

<sup>Created on 2020-10-14 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

(Or: `vaccinations$response <- forcats::fct_rev(vaccinations$response)`)

**Comment by corybrunson** [comment]
Thank you so much @jtr13 for pointing this out! That code traces way back and i guess i never carefully vetted it once i better understood how factors work. I'll get to work on a patch ASAP—there are several other places, including examples, where the error has metastasized.

**Comment by jtr13** [comment]
Sure, thanks again for this awesome package.

**Comment by corybrunson** [comment]
The issue is resolved in commit 99ff7b4023e217321c89c1599269e1c19543f85a, and the website in particular is updated accordingly. Because the code has not been changed, and to avoid overburdening the maintainers, i will wait until the next code patch or feature upgrade to resubmit to CRAN. : )
